src/funcTest/resources/testProjects/multiproject/build-report.gradle

```diff
@@ -42,6 +42,11 @@ subprojects {
             outputFormats = ["HTML","XML"]
             timestampedReports = false
             exportLineCoverage = true
+            reportAggregator {
+                aggregatedTestStrengthThreshold.set(50)
+                aggregatedMutationThreshold.set(40)
+                aggregatedMaxSurviving.set(3)
```

Comment:
@Pfoerd I sat down to merge it and re-reading the changes, I wonder, if they should be named without the `aggregated` prefix (as they are in the `reportAggregator` parent)?

```
reportAggregator {
    testStrengthThreshold.set(50)
    mutationThreshold.set(40)
    maxSurviving.set(3)
}
```

(just asking about your opinion - no need to perform any work on your side - potential changes I would do on my own after the merge)

Reply:
Looks good, I agree that the prefixes are actually not needed when using the nesting syntax) 👍

Reply:
Btw, I've forgotten that the groovy syntax sugar testStrengthThreshold = 50`doesn't work for nested properties :-).